tests/conftest.py

```diff
@@ -15,7 +15,7 @@
 except ImportError:
     pytest_plugins = []
 else:
-    pytest_plugins = ['hypothesis.extra.pytestplugin']
+    pytest_plugins = []
```

Comment:
Is it wanted?

Reply:
no, I have to find a better way of doing this, for some reason this plugin loads twice and then crashes pytest on my machine, but it should definitely not be commited... second time I did this by accident, sorry.